### DISCUSSION TITLE
Autofill of QTH and Locator while editing a QSOs

### DISCUSSION THREAD

**Discussion by m0urs** [magicbug/Cloudlog#1833] [Ideas / Feature Requests]
Would it be possible to have the auto fill of locator and QTH also while editing an existing QSO? E.g. if one forgets to enter the WWFF or SOTA code before saving a QSO and then edits it afterwards, it currently does auto lookup correctly but not auto fill of QTH and Locator.

**Comment by ofadam** [comment]
I've wondered if there could be a "lookup" button on the edit QSO page that would then pull from the specified callbook integration?

**Comment by magicbug** [comment]
Lookup button feels the safest option
